Lab_2/task-description.md

```diff
@@ -0,0 +1,88 @@
+##**LAB 2.**
+
+###**Part 1. Installing and configuring remote OS on VBox.**
+
+1. install some OS, that differs from your host&#39;s one. Better if it will be Ubuntu/Linux. For this you can use provided on EPAM confluence [instructions](https://kb.epam.com/display/EPMLOCALIT/Virtual+Machine+creation+using+Oracle+VM+VirtualBox) about how to install and set up [official EPAM Ubuntu image](https://kb.epam.com/display/EPMLOCALIT/How+to+deploy+EPAM+Ubuntu+image+%28automatically%29+to+a+virtual+machine).
+2. **Win10** deploying Ubuntu OS on VBox **tips** and **troubleshooting** :
+    - System Requirements for Ubuntu 18/20:
+    - 2 GHz dual-core processor;
+    - 4 GiB RAM (but 1 GiB can work);
+    - 25 GB of hard-drive space;
+    - VGA capable of 1024×768 screen resolution;
+    - In case installation takes too long, or at some point screen frozen, or any actions are not performed, stop running session, go to setting and try to increase base memory and processors count, then restart VM;
+    - In some cases, it is helpful to disable **Virtual Machine Platform** Windows feature;
+    - Also, it is possible for testing purposes to use Ubuntu iso from [official site](https://ubuntu.com/download/desktop) instead EPAM&#39;s iso, as last takes too much more time for installation and running, then official one;
```

Comment:
I believe, that we can place this part into our Task#1 which is about installing VM  on a host machine and setting up Linux OS

Reply:
removed